student.rb

```diff
@@ -1,9 +1,13 @@
 require_relative 'person'
+require_relative 'classroom'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
     super(id, age, parent_permission, name)
     @classroom = classroom
+    classroom&.add_student(self)
   end
 
   def play_hooky
```

Comment:
- [ ]  Good job to create Student class, but you still need to improve something on it.  You should have a method `Classroom `that serves as the `'belongs-to'` side (a student belongs to a classroom). For more check on [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)